ols/src/llms/providers/azure_openai.py

```diff
@@ -16,6 +17,19 @@
 logger = logging.getLogger(__name__)
 
 
+token_cache: dict[str, Any] = {
```

Comment:
don't you wanna to use class instead? (just question, it would be ok to use dict there)

Reply:
might be better from typing perspective indeed